src/Screens/components/DropDownMenu.tsx

```diff
@@ -13,78 +13,34 @@ export type DropDownItem = {
   onPress: () => void;
 };
 
-interface Props {
+type Props = {
   items: DropDownItem[];
   testID?: string;
   tintColor?: string;
-}
-
-const DropDown: React.FC<Props> = ({ items, testID, tintColor }) => {
-  const [isOpen, setOpen] = React.useState(false);
-
-  const toggleOpen = () => {
-    setOpen(!isOpen);
-  };
-
-  const itemActionAndCloseModal = (action: () => void) => {
-    action();
-    toggleOpen();
-  };
+  style: any;
+};
 
+const DropDown: React.FC<Props> = ({ items, style }) => {
   return (
-    <>
-      <RN.TouchableHighlight
-        style={styles.kebab}
-        onPress={toggleOpen}
-        underlayColor={colors.faintBackground}
-        testID={testID}
-      >
-        <RN.Image
-          source={require('../images/three-dot-menu.svg')}
-          style={{ tintColor: tintColor ?? colors.white }}
-        />
-      </RN.TouchableHighlight>
-      <RN.Modal visible={isOpen} transparent onRequestClose={toggleOpen}>
-        <RN.TouchableWithoutFeedback onPress={toggleOpen}>
-          <RN.View style={RN.StyleSheet.absoluteFill}>
-            <RN.View style={styles.dropdown}>
-              {items.map((item, index) => (
-                <RN.TouchableHighlight
-                  key={index}
-                  style={styles.button}
-                  underlayColor={colors.lighterBlue}
-                  onPress={() => itemActionAndCloseModal(item.onPress)}
-                >
-                  <Message id={item.textId} style={styles.text} />
-                </RN.TouchableHighlight>
-              ))}
-            </RN.View>
-          </RN.View>
-        </RN.TouchableWithoutFeedback>
-      </RN.Modal>
-    </>
+    <RN.View style={[styles.dropdown, style]}>
+      {items.map((item, index) => (
+        <RN.TouchableOpacity
+          key={index}
```

Comment:
using index as key is bad. Things can break badly if you do this regurarly

Reply:
key defines is something updates or not. With index as key if something swaps from the list, thigns break

Reply:
Added this to card